### PR TITLE
revision fn

### PR DESCRIPTION
**Pull request by cwehmeyer**
fixes 136
fixes 161

### CODE REVIEW

File: manuscript/manuscript.tex [hunk 1]
```diff
@@ -222,6 +224,33 @@ \subsection{Variational approach and TICA}
...
 
+\subsection{Hidden Markov state models}
+
+The estimation of an MSM requires the dynamics between microstates to be Markovian.
+However, in case of a poor discretization or short trajectories,
+it might be hard or even impossible to meet this requirement (as we will see in Notebook~07).
+
+An alternative, which is much less sensitive to poor discretization,
+is to estimate a hidden Markov model (HMM)~\cite{hmm-baum-welch-alg,hmm-tutorial,jhp-spectral-rate-theory,bhmm-preprint}.
+HMMs require the dynamics in some hidden (unobserved) state space to be Markovian.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: should it be "hidden (**i.e.,** unobserved)" ?

File: manuscript/manuscript.tex [hunk 2]
```diff
@@ -222,6 +224,33 @@ \subsection{Variational approach and TICA}
...
+
+To estimate an HMM, we need a spectral gap after the $m^\textrm{th}$ eigenvalue;
+in practice, a timescale separation $t_m \geq 2t_{m+1}$ is sufficient~\cite{pyemma}.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: maybe "require" instead of "need"?

"timescale separation **of**" ?

File: manuscript/manuscript.tex [hunk 3]
```diff
@@ -222,6 +224,33 @@ \subsection{Variational approach and TICA}
...
+\begin{equation}
+\mathbf{P}(k\tau) \approx \bm{\Pi}^{-1} \bm{\chi}^\top \tilde{\bm{\Pi}} \tilde{\mathbf{P}}^k(\tau) \, \bm{\chi}.
+\end{equation}
+Here, the $\bm{\Pi}=\left[ \pi_1,\dots,\pi_n \right]$ is a diagonal matrix of the $n$ microstates' stationary probabilities,
+$\tilde{\bm{\Pi}}=\left[ \tilde{\pi}_1,\dots,\tilde{\pi}_m \right]$ is a diagonal matrix of the $m$ hidden states' stationary probabilities,
+$\tilde{\mathbf{P}}(\tau)$ is a (small) transition matrix between the $m$ hidden states at lag time $\tau$,
+and $\bm{\chi}$ is an $m\times n$-dimensional row-stochastic matrix
+where each row encodes the probabilities of the $n$ microstates to appear in the corresponding hidden state~\cite{noe-proj-hid-msm}.
+
+An HMM estimation usually yields a small model where each hidden state is considered to be metastable and,
```
Comment (brookehus) [inline manuscript/manuscript.tex]: small = small number of states?

File: manuscript/manuscript.tex [hunk 4]
```diff
@@ -222,6 +224,33 @@ \subsection{Variational approach and TICA}
...
+thus, the number of hidden states is a new hyper-parameter which needs to be chosen carefully.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Do we use hyper-parameter or hyperparameter?

Comment (brookehus) [reply manuscript/manuscript.tex]: can you say anything else about choosing it carefully or refer to the notebook or a reference?

File: manuscript/manuscript.tex [hunk 5]
```diff
@@ -296,12 +325,10 @@ \subsection{The PyEMMA workflow}
 Note that the modeler has to select hyper-parameters at most stages throughout the workflow.
 This selection must be done carefully as poor choices make it hard, or even impossible, to build a good MSM.
 
-While there exist automated schemes~\cite{husic-optimized} for cross-validated optimization in the full hyper-parameter space,
-we chose to adopt a sequential approach where only the hyper-parameters of the current stage are optimized.
+While there exist automated schemes~\cite{husic-optimized} for cross-val\-\-i\-dat\-ed optimization in the full hyper-parameter space,
```
Comment (brookehus) [inline manuscript/manuscript.tex]: before \begin{document}, you can add the following:

\hyphenation{val-i-dat-ed}

instead

Comment (cwehmeyer) [reply manuscript/manuscript.tex]: Good point, thanks!

File: manuscript/manuscript.tex (hunk 4 above)
Comment (cwehmeyer) [reply manuscript/manuscript.tex]: > Do we use hyper-parameter or hyperparameter?
We use hyper-parameter.

**Comment by cwehmeyer** [comment]
Thanks for the review, @brookehus.

File: manuscript/manuscript.tex [hunk 6]
```diff
@@ -222,16 +226,35 @@ \subsection{Variational approach and TICA}
...
+$\tilde{\bm{\Pi}}=\left[ \tilde{\pi}_1,\dots,\tilde{\pi}_m \right]$ is a diagonal matrix of the $m$ hidden states' stationary probabilities,
+$\tilde{\mathbf{P}}(\tau)$ is a (small) transition matrix between the $m$ hidden states at lag time $\tau$,
+and $\bm{\chi}$ is an $m\times n$-dimensional row-stochastic matrix
+where each row encodes the probabilities of the $n$ microstates to appear in the corresponding hidden state~\cite{noe-proj-hid-msm}.
+
+An HMM estimation usually yields a model with a small number of (hidden) states
+where each state is considered to be metastable and,
+thus, the number of hidden states is a new hyper-parameter which needs to be chosen carefully (see Notebook~07).
+Given that an HMM is a generalization of an MSM,
+we can compute all physico-chemically relevant quantities available for MSMs, too.
```
Comment (psolsson) [inline manuscript/manuscript.tex]: "Given that an HMM is a generalization of an MSM,
we can compute all physico-chemically relevant quantities available for MSMs, too."
to
"As the HMMs --- like MSMs --- approximate the full phase-space dynamics we can similarly compute physical observables from directly."

**Pull request merged**